pkg/builddocker/build.go

```diff
@@ -44,6 +45,82 @@ func GenerateDockerfile(w io.Writer, env hcl2nix.OCIArtifact, platform string) e
 		return err
 	}
 
+	return nil
+
+}
+
+// ModifyDockerfile modifies the Dockerfile with the specified tag
+func ModifyDockerfile(path, tag string, dev bool) error {
+	var dockerfilePath string
```

Comment:
I think instead of passing path here , we could pass the the data (open the file in cmd/).